src/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { Person } from '../types/Person';
+import { useSearchParams } from 'react-router-dom';
+
+interface PeopleTableProps {
+  people: Person[];
+  selectedSlug?: string;
+  onSort: (field: keyof Person) => void;
+  sortOrder: 'asc' | 'desc' | null;
+  sortField: keyof Person | null;
+}
+
+export const PeopleTable: React.FC<PeopleTableProps> = ({
+  people,
+  selectedSlug,
+  onSort,
+  sortOrder,
+  sortField,
+}) => {
+  const [searchParams] = useSearchParams();
+
+  const findPersonSlug = (
+    name: string | null | undefined,
+    persons: Person[],
+  ): string | undefined => {
+    return persons.find(person => person.name === name)?.slug;
+  };
+
+  const getSortIcon = (field: keyof Person) => {
+    if (sortField === field) {
+      return sortOrder === 'asc' ? (
+        <i className="fas fa-sort-up" />
+      ) : (
+        <i className="fas fa-sort-down" />
+      );
+    }
+
+    return <i className="fas fa-sort" />;
+  };
+
+  const searchParamsString = searchParams.toString(); // Get the current search params as a string
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th onClick={() => onSort('name')}>Name {getSortIcon('name')}</th>
```

Comment:
```suggestion
          <th onClick={() => onSort('name')}>Name{getSortIcon('name')}</th>
```
That is the reason tests fail